src/App.js

```diff
@@ -225,7 +225,7 @@ class App extends Component {
                   className={`${styles.FooterLeft} ${gridStyles.colLg6} ${gridStyles.colMd6} ${gridStyles.colSm6} ${gridStyles.colXs6}`}
                 >
                   <a
-                    href="http://docs.lizard.net/en/latest/introduction.html"
+                    href="http://docs.lizard.net"
```

Comment:
@hoanphungt 
Could you make the link `https://docs.lizard.net`?
(with http**s**)?

Then it is more secure. :)

Reply:
Ah dank je :)